tests/fixture/network.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package fixture
+
+import (
+	"github.com/ava-labs/avalanchego/config"
+	"github.com/ava-labs/avalanchego/tests/fixture/tmpnet"
+)
+
+func NewTmpnetNetwork(owner string, nodes []*tmpnet.Node, subnets ...*tmpnet.Subnet) *tmpnet.Network {
+	return &tmpnet.Network{
+		Owner: owner,
+		Nodes: nodes,
+		DefaultFlags: tmpnet.FlagsMap{
+			config.ProposerVMUseCurrentHeightKey:               true,
+			config.InboundThrottlerVdrAllocSizeKey:             10737418240,
```

Comment:
Also wondering where these defaults come from

Reply:
These come from the existing code. Some of them are arbitrary or determined by iterating to the necessary parameters to make sure AvalancheGo does not overthrottle.